subsys/mpsl/cx/nrf700x/mpsl_cx_nrf700x.c

```diff
@@ -65,6 +65,12 @@ static const struct gpio_dt_spec req_spec     = GPIO_DT_SPEC_GET(CX_NODE, req_gp
 static const struct gpio_dt_spec status0_spec = GPIO_DT_SPEC_GET(CX_NODE, status0_gpios);
```

Comment:
These are now used only in the `mpsl_cx_init` function. To save memory please move these variables to `mpsl_cx_init` as local non-static variables.
```c
struct gpio_dt_spec req_spec     = GPIO_DT_SPEC_GET(CX_NODE, req_gpios);
struct gpio_dt_spec status0_spec = GPIO_DT_SPEC_GET(CX_NODE, status0_gpios);
```